schema/query/fuzz_test.go

```diff
@@ -0,0 +1,9 @@
+package query
```

Comment:
IF only access to the public interface is needed, switch to `package query_test`.